packages/components/src/globals/scss/_theme-tokens.scss

```diff
@@ -248,6 +248,17 @@
   /// @group notification
   $notification-success-background-color: $ibm-color__green-10 !default !global;
 
+  /// @type Color
+  /// @access public
+  /// @group notification
+  $notification-support-01: carbon--theme-custom(
```

Comment:
Wouldn't the default value for this simply be the `$support-01` variable that is already changed per theme?  This seems overly complicated and I'm not understanding why it needs to be.

Reply:
@elizabethsjudd here's the PR for reference

https://github.com/carbon-design-system/carbon-components/pull/2367

Reply:
@tw15egan I've been following that PR/Issue as well but again I don't understand why there needs to be a function.  The value is set to `$red-50` in #2367 so why couldn't this be `$notification-support-01: $red-50 !default`.

I thought the idea of this PR was to allow Carbon users to theme at the component level instead of the global level. For example: $red-50 globally might be `#f00` but for the notification the user wants to use a different hex value so they would then be able to set `$notification-support-01: #f01` without affecting other references to `$red-50` (the hex values are completely made up for demonstration purposes).

Reply:
Also now that I'm thinking about it the variable name `$notification-support-01` doesn't provide clear understanding that this is changing the error notification.  You'd want a semantic variable name so the user understood what they are changing for that component, such as `$notification-error-accent`.